cmd/ef_tests/blockchain/test_runner.rs

```diff
@@ -295,7 +295,7 @@ fn parse_json_file(path: &Path) -> HashMap<String, TestUnit> {
 
 /// Creats a new in-memory store and adds the genesis state
 pub async fn build_store_for_test(test: &TestUnit) -> Store {
-    let store =
+    let mut store =
```

Comment:
can't we avoid making the store mutable? for example by having arcs in the fields of the store